website/tailwind.config.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
   purge: ["./src/**/*.html", "./src/**/*.js", "./src/**/*.tsx"],
   darkMode: "media", // or 'media' or 'class'
+  important: "#tailwind",
```

Comment:
tried this to avoid distortion in docusaurus pages 
I tried to use tailwind only when the component is wrapped with id="tailwind"
but it doesn't seem to work!